core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -64,4 +64,18 @@
       </div>
     </li>
   </ul>
+  <script type="text/ng-template" id="modals/confirmDiscardChanges">    
+    <div class="modal-header">
+      <button type="button" class="close" ng-click="close(false)" data-dismiss="modal" aria-hidden="true">&times;</button>
+      <h4 class="modal-title">Confirmation Required</h4>
+    </div>
+    <div class="modal-body">
+      <p>Are you sure you want to discard the changes?</p>
```

Comment:
Sorry, I forgot to reply to this. We don't need to make this translatable, since we are not translating the editor at the moment.